telemetry/definitions/commonDefinitions.json

```diff
@@ -989,6 +1013,11 @@
             "type": "string",
             "description": "The id of a security finding from a code scan"
         },
+        {
+            "name": "findingsCount",
+            "type": "int",
+            "description": "Number of findings discovered after execution"
```

Comment:
can you be more specific about what kind of execution?

Reply:
Updated it to "Number of findings discovered after executing IAM Policy Checks"